testing/regress/ecl/genjoin.ecl

```diff
@@ -25,8 +25,10 @@ jrec := RECORD,MAXLENGTH(100)
     UNSIGNED1 i;
     STRING3 lstr;
     STRING3 rstr;
-    UNSIGNED1 c;
```

Comment:
(it's ok to check in the resulting test files but should also check in updated genjointest.pl if it IS still valid...)